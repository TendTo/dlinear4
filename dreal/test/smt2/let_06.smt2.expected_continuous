delta-sat with delta = 5e-324 ( > 0)
(model
  (define-fun a () Real 5)
  (define-fun b () Real 0)
)
