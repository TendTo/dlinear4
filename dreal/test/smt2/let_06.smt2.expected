delta-sat with delta = 0.0009999999999999998 ( > 2305843009213693/2305843009213693952)
(model
  (define-fun a () Real 5)
  (define-fun b () Real 0)
)
